### DISCUSSION TITLE
Webforms 4.8 integration : how to do it?

### DISCUSSION THREAD

**Discussion by LuciferSam8086** [App-vNext/Polly#2294] [Q&A]
Hello there, 
I have an old webforms ( aspx ) application. 
Is it possible to create a global pipeline for a rate limiter in global.asax and then use it in certain pages / controllers ? Or do you have other suggestions? 

We're using .NET 4.8. 

Thank you!

**Comment by peter-csala** [comment]
Polly targets .NET Standard 1.1 so, .NET 4.8 is certainly supported. But it is worth mentioning that Polly provides only a [thin layer on top of](https://www.pollydocs.org/strategies/rate-limiter.html) the [System.Threading.RateLimiting](https://www.nuget.org/packages/System.Threading.RateLimiting). If you want to use only just the rate limiter strategy then I would suggest to use directly the `System.Threading.RateLimiting` package instead of `Polly`.
